pkg/collector/metrics_test.go

```diff
@@ -0,0 +1,37 @@
+package collector
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"fmt"
+)
+
+var _ = Describe("Test Collector Unit", func() {
+	It("Check feature values", func() {
+		Expect(len(uintFeatures)).Should(BeNumerically(">", 0))
+		Expect(len(collectedLabel)).Should(BeNumerically(">", 0))
+		Expect(len(podEnergyLabels)).Should(BeNumerically(">", 0))
+		fmt.Printf("%v\n%v\n%v\n", uintFeatures, collectedLabel, podEnergyLabels)
+	})
+
+
+	It("Check convert values", func() {
+
+		podEnergy := &PodEnergy{
+			PodName: "podA",
+			Namespace: "default",
+			AggEnergyInCore: 10,
+			CgroupFSStats: map[string]*UInt64Stat{
+				CPU_USAGE_TOTAL_KEY: &UInt64Stat{
+					Curr: SAMPLE_CURR,
```

Comment:
where is `SAMPLE_CURR` defined?

Reply:
My bad. It is defined in collector_test.go.
I will redefine it separately in metric_test.go.